build/params_2k.go

```diff
@@ -39,6 +39,8 @@ var UpgradeNorwegianHeight = abi.ChainEpoch(40)
 
 var UpgradeActorsV4Height = abi.ChainEpoch(45)
 
+var UpgradeHyperdriveHeight = abi.ChainEpoch(50)
```

Comment:
ActorsV5? I mean, I know hyperdrive is nicer... but it's kind of inconsistent.

That or we can go back and name all the other heights. I'd actually prefer to do it that way.

Reply:
Yeah, I'll rename the old ones -- the inconsistency was bothering me too. I think we want to go with Hyperdrive for now (and names from this point on).

Reply:
`UpgradeHyperspaceBypassHeight`? We can totally print some bad poetry at the time of the upgrade.